File: app/src/main/java/ir/beigirad/admediation/MainActivity.java
package ir.beigirad.admediation;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import ir.beigirad.admediation.example.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnInitializer.setOnClickListener(v -> AdMediation.initialize());
    }
}

File: app/src/main/java/ir/beigirad/admediation/example/MainActivity.kt
package ir.beigirad.admediation.example

import android.app.Activity
import android.os.Bundle
import ir.beigirad.admediation.AdMediation

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        AdMediation.initialize()
    }
}